Server/mods/deathmatch/logic/CStaticFunctionDefinitions.cpp

```diff
@@ -8235,6 +8235,24 @@ bool CStaticFunctionDefinitions::StopObject(CElement* pElement)
     return false;
 }
 
+bool CStaticFunctionDefinitions::BreakObject(CElement* pElement)
+{
+    RUN_CHILDREN(BreakObject(*iter));
```

Comment:
Add min client version check.
Server should not call this when the minimal client version doesn't support this event to avoid desync problems.

Reply:
You mean adding min. version in CResourceChecker?